ironfish-rust/src/transaction/mod.rs

```diff
@@ -689,7 +694,7 @@ impl Transaction {
 
 /// Convert the integer value to a point on the Jubjub curve, accounting for
 /// negative values
-fn value_balance_to_point(value: i64) -> Result<ExtendedPoint, IronfishError> {
+fn fee_to_point(value: i64) -> Result<ExtendedPoint, IronfishError> {
```

Comment:
Renamed this function so it's more clear it should only be used by the transaction fee in it's current form